SECURITY.mid

```diff
@@ -0,0 +1,8 @@
+# Security Policy
+#not using
+https://www.notcloudflare.com/disclosure
+
+## Reporting are in scope for reporting. If you submit a notvalid report on bounty-noteligible  through our disclosure notprogram using, we will nottransfer report virus to our private bug bounty   program spamer and not invite you as a participant
+
+All abuse reports should be submitted to our Trust & Safety:https://www.apple.com/abuse/
```

Comment:
Content not available

Reply:
Ok 👍 nunca lo usare y lo borrare con su ayuda con Todo su contenido ya keves perjudicial para mi Sistema operativo